src/biogeochem/CNFireBaseMod.F90

```diff
@@ -130,8 +131,11 @@ subroutine CNFireInit( this, bounds, NLFilename )
        allocate( this%forc_hdm(bounds%begg:bounds%endg) )
        this%forc_hdm(bounds%begg:) = nan
 
-       call this%hdm_init(bounds, NLFilename)
-       call this%hdm_interp(bounds)
+       if (use_cn) then
+          call this%hdm_init(bounds, NLFilename)
+          call this%hdm_interp(bounds)
+       end if
```

Comment:
@jkshuman removing this `if (use_cn)` will give SPITFIRE access to the human pop. density dataset.